<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TicTacToeGame</class>
 <widget class="QDialog" name="TicTacToeGame">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>555</width>
    <height>399</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Tic Tac Toe</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton[cell=true]{
font: 50px &quot;Verdana&quot;;
min-height: 100px;
max-height: 100px;
min-width: 100px;
max-width: 100px;
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QGridLayout" name="gridBoard">
         <property name="spacing">
          <number>0</number>
         </property>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="styleSheet">
          <string notr="true">QWidget
{
	background-color: #00111d;
	color: #ffffff;
	border-color: #000000;

}</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="reset">
            <property name="styleSheet">
             <string notr="true">QPushButton
{
	background-color:#e3e3e3;
	color: #1323ff;
	border-style: solid;
	border-radius:15px;
	border-color: #000000;
	font-weight : bold;
	padding: 13px;

}


QPushButton::hover
{
	background-color: #3dffaf;
	color: #000000;
	border-style: solid;
	border-color: #000000;
	font-weight : bold;
	padding: 13px;

}


QPushButton::pressed
{
	background-color: #3dc8af;
	color: #000000;
	border-style: solid;
	border-color: #000000;
	font-weight : bold;
	padding: 13px;

}</string>
            </property>
            <property name="text">
             <string>New Game</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="font">
             <font>
              <family>Segoe UI Semibold</family>
              <pointsize>13</pointsize>
             </font>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="back">
            <property name="styleSheet">
             <string notr="true">QPushButton
{
	background-color:#e3e3e3;
	color: #1323ff;
	border-style: solid;
	border-radius:15px;
	border-color: #000000;
	font-weight : bold;
	padding: 13px;

}


QPushButton::hover
{
	background-color: #3dffaf;
	color: #000000;
	border-style: solid;
	border-color: #000000;
	font-weight : bold;
	padding: 13px;

}


QPushButton::pressed
{
	background-color: #3dc8af;
	color: #000000;
	border-style: solid;
	border-color: #000000;
	font-weight : bold;
	padding: 13px;

}</string>
            </property>
            <property name="text">
             <string>Back to title screen</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
